### PR TITLE
fix: skip DBs with read-only schema

### PR DESCRIPTION
**Pull request by leon0399**
## 🔍 What was changed

Fixes errors when, during migrations, an error occurred with read-only schemas

## 🤔 Why?

We're migrating some services to a new tech stack, and as part of this process, we want to use one of DBs in read-only mode

## 📝 Checklist

- How was this tested:
  - [x] Tested manually
  - [ ] Unit tests added

### CODE REVIEW

**Review by roxblnfk — APPROVED** [review]
Looks good.
Is it possible to add a test here?

**Comment by leon0399** [comment]
Not sure how to approach it atm
By the looks of things, current tests expect to have a single DB driver per test; technically, this test requires a separate driver. 

I don't want to introduce big changes to the testing approach without approval, so if maintainers can take a look and propose some ideas, that would be great!

**Pull request merged**
